src/iosched/unified.c

```diff
@@ -514,8 +564,7 @@ ssize_t unified_read(struct dentry *d, char *buf, size_t size, off_t offset, voi
 		goto out;
 	releaseread_mrsw(&priv->vol->lock);
 
-	ltfs_mutex_lock(&d->iosched_lock);
```

Comment:
Are you sure you want to remove this? `iosched_lock` is needed to protect access to dpr->requests. If you are sure about removing it, then please note that you still have calls to ltfs_mutex_unlock() in this function.

Reply:
No, it is my mistake. Fixed in my branch.